src/main/java/io/jenkins/plugins/okhttp/api/internals/JenkinsProxyAuthenticator.java

```diff
@@ -47,4 +48,18 @@ public Request authenticate(@Nullable Route route, Response response) throws IOE
 
         return null;
     }
+
+    private boolean isAuthenticationSchemeSupported(@NonNull final String proxyAuthenticateHeader) {
+        final String lowerCaseHeader = proxyAuthenticateHeader.toLowerCase();
```

Comment:
lower case of `BASIC` is not always `basic` and so this code will not always work.

please ensure the use of `Locale.ROOT` and not rely on the locale that Jenkins is running in.

Reply:
Fixed